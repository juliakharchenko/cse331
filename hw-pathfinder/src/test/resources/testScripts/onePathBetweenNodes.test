# Find a path with Dijkstra's algorithm between
# two nodes where only one path exists between them.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C

AddEdge simpleGraph A B 10.0
AddEdge simpleGraph B C 5.0

FindPath simpleGraph A B
FindPath simpleGraph B C