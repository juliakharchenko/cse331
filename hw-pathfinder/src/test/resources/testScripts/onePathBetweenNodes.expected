# Find a path with Dijkstra's algorithm between
# two nodes where only one path exists between them.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph

added edge 10.000 from A to B in simpleGraph
added edge 5.000 from B to C in simpleGraph

path from A to B:
A to B with weight 10.000
total cost: 10.000
path from B to C:
B to C with weight 5.000
total cost: 5.000